frontend/packages/ui/src/styles/Dark/variables.css

```diff
@@ -161,6 +161,7 @@
   --spacing-10: 40px;
   --spacing-12: 48px;
   --spacing-14: 56px;
+  --spacing-16: 64px;
```

Comment:
This custom property was removed in https://github.com/liam-hq/liam/commit/423066159cebfa8c1ac0e7ea6e07c770529e6375

Reply:
I'm not sure if it should be removed, but this is the simplest way to fix.